### PR TITLE
Add install instructions for Fedora

### PR DESCRIPTION
**Pull request by tijptjik**

### CODE REVIEW

**Comment by smathles** [comment]
Great to see another Fedora enjoyer! I have actually also done a similar project on a personal repo that I've never made a PR for [link](https://github.com/smathles/HyprPanelSmathles/edit/master/README.md). I see some things you implemented that I either missed or did more efficiently than I did (grimblast using the copr repo and I missed bluez-tools).

I note we had different approaches to install sass and matugen: 
- I used `sudo dnf install nodejs; sudo npm install -g sass` where you had `bun install -g sass`. As we should already have bun installed, I prefer your way. 
- For matugen, I used [this copr repo](https://copr.fedorainfracloud.org/coprs/heus-sueh/packages/package/matugen/), as it was up to date with the latest matugen and install size is only 9.6M as opposed to installing cargo first and using it to install matugen (250M if you don't already have rust/cargo packages installed). Of course, using copr repos comes with the risk that one day it may not be maintained (just like the AUR), so depends on what @Jas-SinghFSU's preference is there. Slight caveat to this method is that this copr repo also packages swww... so it would be best to install everything else from the first repo first and then enable this one and install matugen to avoid any package ambiguity (FYI I did this and it works fine on my system. Would you be able to try it and see if there's any conflicts?).

Additionally I think the optional python/gpustat/pywal section is missing? I know not everyone might want it but it's still good to put down explicitly imo.

File: README.md [hunk 1]
```diff
@@ -57,16 +57,43 @@ pywal
...
-Arch (AUR):
+AUR:
 ```bash
 yay -S grimblast-git gpu-screen-recorder hyprpicker matugen-bin python-gpustat aylurs-gtk-shell-git
 ```
 
+### Fedora
+
+DNF - with solopasha's [hyprland COPR](https://copr.fedorainfracloud.org/coprs/solopasha/hyprland/)
+```bash
```
Comment (smathles) [inline README.md:75]: Add sudo to the dnf install instructions

File: README.md [hunk 2]
```diff
@@ -57,16 +57,43 @@ pywal
...
+dnf copr enable solopasha/hyprland
+dnf install pipewire libgtop2 bluez bluez-tools grimblast hyprpicker btop NetworkManager cargo wl-clipboard swww brightnessctl gnome-bluetooth aylurs-gtk-shell
+```
```
Comment (smathles) [inline README.md:78]: Optionally remove cargo section and change dnf section to:
> DNF - with solopasha's [hyprland COPR](https://copr.fedorainfracloud.org/coprs/solopasha/hyprland/) and heus-sueh's [matugen COPR](https://copr.fedorainfracloud.org/coprs/heus-sueh/packages/packages/)
```bash
dnf copr enable solopasha/hyprland
dnf install pipewire libgtop2 bluez bluez-tools grimblast hyprpicker btop NetworkManager cargo wl-clipboard swww brightnessctl gnome-bluetooth aylurs-gtk-shell
dnf copr enable heus-sueh/packages
sudo dnf install matugen
```
Subject to testing + @Jas-SinghFSU's preference.

File: README.md [hunk 3]
```diff
@@ -57,16 +57,43 @@ pywal
...
+
+flatpak:
+```bash
+flatpak install flathub --system com.dec05eba.gpu_screen_recorder
+```
+
+cargo:
+```bash
+cargo install matugen
```
Comment (smathles) [inline README.md:87]: Optional: see previous comment

File: README.md [hunk 4]
```diff
@@ -57,16 +57,43 @@ pywal
...
+```
+
+bun:
+```bash
+bun install -g sass
+```
+
```
Comment (smathles) [inline README.md:94]: Requesting to add:

> Python (Optional):
> ```bash
> sudo dnf install python python3-pip; pip install gpustat pywal
> ```

File: README.md (hunk 2 above)
Comment (smathles) [inline README.md:78]: Optionally remove cargo section and change dnf section to:
> DNF - with solopasha's [hyprland COPR](https://copr.fedorainfracloud.org/coprs/solopasha/hyprland/) and heus-sueh's [matugen COPR](https://copr.fedorainfracloud.org/coprs/heus-sueh/packages/packages/)
```bash
dnf copr enable solopasha/hyprland
dnf install pipewire libgtop2 bluez bluez-tools grimblast hyprpicker btop NetworkManager cargo wl-clipboard swww brightnessctl gnome-bluetooth aylurs-gtk-shell
dnf copr enable heus-sueh/packages
sudo dnf install matugen
```
Subject to testing + @Jas-SinghFSU's preference.

File: README.md [hunk 5]
```diff
@@ -57,16 +57,43 @@ pywal
...
+AUR:
 ```bash
 yay -S grimblast-git gpu-screen-recorder hyprpicker matugen-bin python-gpustat aylurs-gtk-shell-git
 ```
 
+### Fedora
+
+DNF - with solopasha's [hyprland COPR](https://copr.fedorainfracloud.org/coprs/solopasha/hyprland/)
+```bash
+dnf copr enable solopasha/hyprland
```
Comment (smathles) [inline README.md:76]: Add `sudo` to the dnf and flatpak install instructions to be consistent with arch install instructions (yay doesn't need sudo).

**Review by smathles — CHANGES_REQUESTED** [review]
Some changes I think would be good (add python section), others are very subjective (get @Jas-SinghFSU to approve). Interested to hear your thoughts :pray:

**Comment by smathles** [comment]
Ooh wait missed one. `gvfs` is now a dependency as well (recentish commit) so that should be added to the dnf install list (yes, it's in the fedora repos, no stress).

**Comment by tijptjik** [comment]
Great! Thanks for the feedback @smathles! I've made the following changes after syncing with `HEAD` on `master` :

- Add subsection for `COPR` setup
- Add Priority to `solopasha/hyprland` to avoid conflicts in the `swww` package.
- Remove `cargo` from DNF instructions
- Remove `cargo` instructions
- Add `power-profiles-daemon and `gvfs` to DNF instructions
- Add requested python instructions
- Add `sudo` to the commands

**Comment by smathles** [comment]
Looks awesome. Absolutely nothing I'd change!

@Jas-SinghFSU are you happy with the way this looks in the README.md file? I know you'll also have to update your webpage for the wiki at some point. 

Looks great to me though.

**Comment by Jas-SinghFSU** [comment]
Lgtm

**Comment by Jas-SinghFSU** [comment]
Ty for the contribution :heart:

**Pull request merged**